src/plugins/intel_gna/src/transformations/matops_to_dwsc.cpp

```diff
@@ -0,0 +1,261 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#include "matops_to_dwsc.hpp"
+
+#include <ngraph/opsets/opset1.hpp>
+#include <ngraph/pass/graph_rewrite.hpp>
+#include <ngraph/pattern/op/or.hpp>
+#include <ngraph/pattern/op/wrap_type.hpp>
+#include <ngraph/rt_info.hpp>
+#include <openvino/cc/ngraph/itt.hpp>
+
+namespace ov {
+namespace intel_gna {
+namespace pass {
+
+// Check if the previous node is convolution, groupconvolution or matmul followed by add
```

Comment:
Shouldn't it be something like: "Check if the node is NOT convolution, groupconvolution, nor matmul"?
Where the "followed by add" is checked?

Reply:
Fixed the code. 
![image](https://user-images.githubusercontent.com/6750072/224183587-1c808388-8bab-4e72-a73d-d661b2483c4e.png)

Here intention is, if an add node is encountered, then check if its parent node is a convolution, groupconvolution or matmul in which case the add can be fused and the decomposition to DWSC is not required.